examples/src/main/scala/org/apache/spark/examples/graphx/LiveJournalPageRank.scala

```diff
@@ -17,11 +17,6 @@
 
 package org.apache.spark.examples.graphx
 
-import org.apache.spark.SparkContext._
```

Comment:
Yes, I think it is OK.
The description of the parameter will be output by the Analytics.Main, looks like:
```
JackydeMacBook-Pro:spark jackylee$ bin/run-example graphx.LiveJournalPageRank ../data/Wiki-Vote.txt
Set the number of edge partitions using --numEPart.
Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
JackydeMacBook-Pro:spark jackylee$ 
```